src/avram/query_builder.cr

```diff
@@ -220,23 +228,30 @@ class Avram::QueryBuilder
   end
 
   private def joined_wheres_queries
-    if @wheres.any? || @raw_wheres.any?
-      statements = @wheres.map do |sql_clause|
-        if sql_clause.is_a?(Avram::Where::NullSqlClause)
-          sql_clause.prepare
-        else
-          sql_clause.prepare(next_prepared_statement_placeholder)
-        end
+    if @wheres.any?
+      statements = @wheres.flat_map do |where_clause|
+        where_clause_sql =
+          case where_clause
+          when Avram::Where::Raw
+            where_clause.to_sql
```

Comment:
I wonder if we would want to rename `prepare` to `to_sql` to align with `Avram::Where::Raw`? It may also better clarify the method's behavior.